ci/cpu/cugraph/upload-anaconda.sh

```diff
@@ -1,6 +1,16 @@
 #!/bin/bash
+# Copyright (c) 2018-2020, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
 #
-# Adopted from https://github.com/tmcdonell/travis-scripts/blob/dfaac280ac2082cd6bcaba3217428347899f2975/update-accelerate-buildbot.sh
```

Comment:
Is this deletion intentional?  Seems like if we adopted this file from somewhere we should maintain that history.

Reply:
I checked the reference and it looked like a completely different file, so I thought it might be an old copy/paste error?  I can certainly add it back in though if we think that's best...thoughts?

Reply:
Makes sense.  I also checked the reference.  Any resemblance this may have had at one time to the reference is certainly long gone.

Reply:
Maybe @raydouglass has a better perspective on this one?

Reply:
That's leftover from our old travis-ci build process. It's not relevant anymore.